src/tcgwars/logic/impl/gen4/DiamondPearlPromos.groovy

```diff
@@ -651,7 +654,7 @@ public enum DiamondPearlPromos implements LogicCardInfo {
             delayedA {
               before BEGIN_TURN, {
                 if (my.all.find {it.name == "Darkrai" }) {
-                  heal 10, self, POKEBODY
```

Comment:
Doesn't the IDE yell at you that `Source.` isn't needed after adding the import on line 8?

Reply:
Not sure why but removing the `Source.` causes POKEBODY to still be interpreted as an EffectType instead of a Source despite EffectType not being a legal argument.

Reply:
Oh, I see EffectType is also wildcarded there, and it is done first. Yeah that makes sense.